libr/anal/flirt.c

```diff
@@ -1037,7 +1037,7 @@ static bool parse_tree(RFlirt *f, RFlirtNode *root_node) {
 
 #if DEBUG
 
-#define PRINT_ARCH(define, str) if (arch == define) { eprintf (" %s", str); return; }
+#define PRINT_ARCH(define, str) if (arch == define) { R_LOG_ERROR (" %s", str); return; }
```

Comment:
this is not an error. its listing archs, and it shuold be constructing a string instead of peirnting it directly but keep the eprintf for now